dataactcore/migrations/versions/88e9b634ca1a_create_published_award_financial_.py

```diff
@@ -0,0 +1,96 @@
+    sa.Column('awardee_or_recipient_uniqu', sa.Text(), nullable=True),
+    sa.Column('awarding_agency_code', sa.Text(), nullable=True),
+    sa.Column('awarding_office_code', sa.Text(), nullable=True),
+    sa.Column('awarding_sub_tier_agency_c', sa.Text(), nullable=True),
+    sa.Column('award_modification_amendme', sa.Text(), nullable=True),
+    sa.Column('business_funds_indicator', sa.Text(), nullable=True),
+    sa.Column('business_types', sa.Text(), nullable=True),
+    sa.Column('cfda_number', sa.Text(), nullable=True),
+    sa.Column('correction_late_delete_ind', sa.Text(), nullable=True),
+    sa.Column('face_value_loan_guarantee', sa.Numeric(), nullable=True),
+    sa.Column('fain', sa.Text(), nullable=True),
+    sa.Column('federal_action_obligation', sa.Numeric(), nullable=True),
+    sa.Column('fiscal_year_and_quarter_co', sa.Text(), nullable=True),
+    sa.Column('funding_agency_code', sa.Text(), nullable=True),
+    sa.Column('funding_office_code', sa.Text(), nullable=True),
+    sa.Column('funding_sub_tier_agency_co', sa.Text(), nullable=True),
+    sa.Column('legal_entity_address_line1', sa.Text(), nullable=True),
+    sa.Column('legal_entity_address_line2', sa.Text(), nullable=True),
+    sa.Column('legal_entity_address_line3', sa.Text(), nullable=True),
+    sa.Column('legal_entity_country_code', sa.Text(), nullable=True),
+    sa.Column('legal_entity_foreign_city', sa.Text(), nullable=True),
+    sa.Column('legal_entity_foreign_posta', sa.Text(), nullable=True),
+    sa.Column('legal_entity_foreign_provi', sa.Text(), nullable=True),
+    sa.Column('legal_entity_zip5', sa.Text(), nullable=True),
+    sa.Column('legal_entity_zip_last4', sa.Text(), nullable=True),
+    sa.Column('non_federal_funding_amount', sa.Numeric(), nullable=True),
+    sa.Column('original_loan_subsidy_cost', sa.Numeric(), nullable=True),
+    sa.Column('period_of_performance_curr', sa.Text(), nullable=True),
+    sa.Column('period_of_performance_star', sa.Text(), nullable=True),
+    sa.Column('place_of_performance_code', sa.Text(), nullable=True),
+    sa.Column('place_of_performance_congr', sa.Text(), nullable=True),
+    sa.Column('place_of_perform_country_c', sa.Text(), nullable=True),
+    sa.Column('place_of_performance_forei', sa.Text(), nullable=True),
+    sa.Column('place_of_performance_zip4a', sa.Text(), nullable=True),
+    sa.Column('record_type', sa.Integer(), nullable=True),
+    sa.Column('sai_number', sa.Text(), nullable=True),
+    sa.Column('uri', sa.Text(), nullable=True),
+    sa.PrimaryKeyConstraint('published_award_financial_assistance_id'),
+    sa.UniqueConstraint('awarding_sub_tier_agency_c', 'award_modification_amendme', 'fain', 'uri', name='uniq_award_mod_sub_tier_fain_uri')
+    )
+    op.create_index(op.f('ix_published_award_financial_assistance_award_modification_amendme'), 'published_award_financial_assistance', ['award_modification_amendme'], unique=False)
```

Comment:
Yes, I didn't touch the generation file in any way. The only part I modified was the model.